ign_ros2_control/src/ign_ros2_control_plugin.cpp

```diff
@@ -325,14 +320,14 @@ void IgnitionROS2ControlPlugin::Configure(
     argv.push_back(reinterpret_cast<const char *>(arg.data()));
   }
 
+  // Create a default context, if not already
   if (!rclcpp::ok()) {
     rclcpp::init(static_cast<int>(argv.size()), argv.data());
-    std::string node_name = "ignition_ros_control";
-    if (!controllerManagerPrefixNodeName.empty()) {
-      node_name = controllerManagerPrefixNodeName + "_" + node_name;
-    }
-    this->dataPtr->node_ = rclcpp::Node::make_shared(node_name);
   }
+
+  std::string node_name = "ignition_ros_control";
```

Comment:
```suggestion
  std::string node_name = "gz_ros2_control";
```

let's use the future naming scheme straight away